spec/std/env_spec.cr

```diff
@@ -172,4 +172,75 @@ describe "ENV" do
   ensure
     ENV.delete("FOO")
   end
+
+  it "clears and replaces" do
+    # Add a dummy value so that post replace we can ensure
+    # at least one variable exists in the ENV.
+    ENV["TEST_DUMMY_VAR"] = "dummy_value"
+    original_env = ENV.to_h
+    ENV.clear
+    ENV.should be_empty
+    ENV.replace original_env
+    ENV.should_not be_empty
+    ENV.to_h.should eq original_env
+  ensure
+    ENV.delete("TEST_DUMMY_VAR")
+  end
```

Comment:
I think this spec should be split into two.

This only tests `#replace` with an empty environment. For that test to succeed, `#replace` could just be `#merge`. There should be a test that replaces a populated environment.